build/Publish-LocalCompatModule.ps1

```diff
@@ -38,9 +38,28 @@ if($moduleName -eq 'Entra'){
     Publish-Module -Name Microsoft.Graph.Authentication -RequiredVersion $content.destinationModuleVersion -Repository (Get-LocalPSRepoName) -Force -Verbose
 }
 
-# Publish Graph PowerShell modules (e.g Microsoft.Graph.User) to the the Local gallery.
+# Publish Graph PowerShell modules (e.g Microsoft.Entra.Users) to the the Local gallery.
```

Comment:
This step deals with Graph PowerShell modules.
There is another step below that deals with publishing Entra submodules